common/src/main/java/org/apache/sedona/common/geography/Constructors.java

```diff
@@ -264,4 +264,271 @@ private static Geometry collectionToGeom(Geography g, GeometryFactory gf) {
     }
     return gf.createGeometryCollection(gs);
   }
+
+  public static Geography geomToGeography(Geometry geom) throws IOException {
+    if (geom == null) {
+      return null;
+    }
+    String type = geom.getGeometryType();
+    if ("Point".equalsIgnoreCase(type)) {
+      return pointToGeog(geom);
+    } else if ("MultiPoint".equalsIgnoreCase(type)) {
+      return mPointToGeog(geom);
+    } else if ("LineString".equalsIgnoreCase(type)) {
+      return lineToGeog(geom);
+    } else if ("MultiLineString".equalsIgnoreCase(type)) {
+      return mLineToGeog(geom);
+    } else if ("Polygon".equalsIgnoreCase(type)) {
+      return polyToGeog(geom);
+    } else if ("MultiPolygon".equalsIgnoreCase(type)) {
+      return mPolyToGeog(geom);
+    } else if ("GeometryCollection".equalsIgnoreCase(type)) {
+      return geomCollToGeog(geom);
+    } else {
```

Comment:
I prefer directly using `instanceof` to check the geometry type. Reference: https://github.com/apache/sedona/blob/658bf60548be0fa3c60ea8b0a9637dabab4b30bc/common/src/main/java/org/apache/sedona/common/geometrySerde/GeometrySerializer.java#L42-L59

The order of the checks is important. We need to check for GeometryCollection last because Multi geometries are subclasses of GeometryCollection.